### DISCUSSION TITLE
Github stolen remotley??

### DISCUSSION THREAD

**Discussion by Trizkitz** [community/community#40911] [Other Feature Feedback, Questions, & Ideas] [answered]
I think this Happend to me !! could anyone help me !! I need help pending to need some extra help since it's just me on my own trying to constantly update among another apps recover my git hub please and thanks 🙏

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Trizkitz 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms.  I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
